x-pack/plugins/security_solution/public/app/deep_links/index.ts

```diff
@@ -158,25 +154,22 @@ export const securitySolutionsDeepLinks: SecuritySolutionDeepLink[] = [
         title: ALERTS,
         path: ALERTS_PATH,
         navLinkStatus: AppNavLinkStatus.visible,
+        order: 9001,
         keywords: [
           i18n.translate('xpack.securitySolution.search.alerts', {
             defaultMessage: 'Alerts',
           }),
         ],
-        searchable: true,
```

Comment:
not needed, having `navLinkStatus: AppNavLinkStatus.visible` sets searchable to `true` implicitly